hack/components/bump-knmstate.sh

```diff
@@ -5,10 +5,11 @@ set -xeo pipefail
 source hack/components/yaml-utils.sh
 source hack/components/git-utils.sh
 
+echo kubernetes-nmstate
 NMSTATE_URL=$(yaml-utils::get_component_url nmstate)
 NMSTATE_COMMIT=$(yaml-utils::get_component_commit nmstate)
 NMSTATE_REPO=$(yaml-utils::get_component_repo ${NMSTATE_URL})
-NMSTATE_PATH=${GOPATH}/src/${NMSTATE_REPO}
+NMSTATE_PATH=$(pwd)/_components/go/src/${NMSTATE_REPO}
```

Comment:
Why do you need this under go/src ?

Reply:
the script needs to clone the repo in order to get the tag and also maybe other stuff like the manifests.
using `${GOPATH}/src/${NMSTATE_REPO}` will change my working kubernetes-nmstate repo which I don't want (luckily I didn't have any unpushed code there ^^ ). 

In the past this was handled on bump.components.sh where we would manually change the GOPATH.. 
but I since we're really using go here then I just changed it manually to `$(pwd)/_components/go/src/${NMSTATE_REPO}`

Reply:
What I mean is that you don't need to do any reference to go you can clone it even at a temporal dir with `mktmp -d` and remove that dir at the end of the scripts some signal catcher.